package.json

```diff
@@ -11,104 +13,27 @@
     "bugs": {
         "url": "https://github.com/hey-api/openapi-ts/issues"
     },
-    "license": "MIT",
-    "keywords": [
-        "openapi",
-        "swagger",
-        "generator",
-        "typescript",
-        "javascript",
-        "codegen",
-        "yaml",
-        "json",
-        "fetch",
-        "xhr",
-        "axios",
-        "angular",
-        "node"
-    ],
-    "main": "./dist/node/index.js",
-    "types": "./dist/node/index.d.ts",
-    "bin": {
-        "openapi-ts": "bin/index.js"
-    },
-    "files": [
-        "bin",
-        "dist"
-    ],
     "scripts": {
-        "build-bundle": "rollup --config rollup.config.ts --configPlugin typescript",
-        "build-types-check": "tsc --project tsconfig.check.json",
-        "build-types-roll": "rollup --config rollup.dts.config.ts --configPlugin typescript && rimraf temp",
-        "build-types-temp": "tsc --emitDeclarationOnly --outDir temp -p src/node",
-        "build-types": "run-s build-types-temp build-types-roll build-types-check",
-        "build": "run-s clean build-bundle build-types",
-        "clean": "rimraf dist test/generated test/e2e/generated coverage node_modules/.cache",
-        "dev": "rimraf dist && npm run build-bundle -- --watch",
-        "lint:fix": "eslint . --fix",
-        "lint": "eslint .",
-        "prepublishOnly": "npm run build",
-        "test:coverage": "vitest run --config vitest.config.unit.ts --coverage",
-        "test:e2e": "vitest run --config vitest.config.e2e.ts",
-        "test:sample": "node test/sample.cjs",
-        "test:update": "vitest watch --config vitest.config.unit.ts --update",
-        "test:watch": "vitest watch --config vitest.config.unit.ts",
-        "test": "vitest run --config vitest.config.unit.ts",
-        "typecheck": "tsc --noEmit",
+        "build": "npm run build --workspaces --if-present",
+        "lint:fix": "npm run lint:fix --workspaces --if-present",
+        "lint": "npm run lint --workspaces --if-present",
+        "test:coverage": "npm run test:coverage --workspaces --if-present",
+        "test:e2e": "npm run test:e2e --workspaces --if-present",
+        "test:update": "npm run test:update --workspaces --if-present",
+        "test": "npm run test --workspaces --if-present",
+        "typecheck": "npm run typecheck --workspaces --if-present",
+        "openapi-ts": "npm run --workspace packages/openapi-ts --",
```

Comment:
NOTE: this is used if you want to run script for this package, ex: `npm run openapi-ts test:update`. The above scripts are common scripts used in CI to run in all packages